lib/LedgerSMB/Sysconfig.pm

```diff
@@ -525,6 +520,27 @@ sub check_permissions {
     }
 }
 
+# if you have latex installed set to 1
+###TODO-LOCALIZE-DOLLAR-AT
+our $latex = 0;
+
+
+sub override_defaults {
+    # Check Latex
```

Comment:
This line needs a `local $@` line before it to make sure the error value gets correctly restored.